ssh-pkcs11-client.c

```diff
@@ -286,6 +445,33 @@ pkcs11_start_helper(void)
 		error("socketpair: %s", strerror(errno));
 		return (-1);
 	}
+#ifdef WINDOWS
+	if (posix_spawn_file_actions_init(&actions) != 0) {
+		error_f("posix_spawn_file_actions_init failed");
+		goto out;
+	}
+	actions_inited = 1;
+	if (posix_spawn_file_actions_adddup2(&actions, pair[1],
+		STDIN_FILENO) != 0 ||
+		posix_spawn_file_actions_adddup2(&actions, pair[1],
+			STDOUT_FILENO) != 0) {
+		error_f("posix_spawn_file_actions_adddup2 failed");
+		goto out;
+	}
+
+	av[0] = helper;
+	av[1] = verbosity;
+	av[2] = NULL;
+	HANDLE user_token = NULL;
+
+	if ((user_token = get_current_user_token()) == NULL ||
```

Comment:
Please have a error log if get_current_user_token() fails to return the user token

Reply:
With the latest commit, we no longer use `get_current_user_token()`